File: Workshop 2/The Happy Pirate/src/View/iMenu.java
package View;


import Model.Boat;
import Users.Member;

/**
 * Created by henry on 2016-09-20.
 */
public interface iMenu {

    boolean AuthenticateField(); // Login fields
    String[] LoginField();
    void RegistrationField();

    void MemberMenu();
    void SecretaryMenu();
    void TreasurerMenu();

    void BoatMenu();

}

File: Workshop 2/The Happy Pirate C#/YatchClub/YatchClub/View/iMenu.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using YatchClub.Model;
using YatchClub.Users;

namespace YatchClub.View
{
    interface iMenu
    {
        string AuthenticateField(); // Login fields
        string[] LoginField();      // username and password
        string RegistrationField();

        string MemberField(Member member);
        string SecretaryField(Secretary secretary);
        string TreasurerField(Treasurer treasurer);

        string BoatField(Boat boat);
    }
}
